{"uid":"15d795e7090f9b9e","name":"test_contact_message_form","fullName":"tests.test_contact_message.TestContactMessageForm#test_contact_message_form","historyId":"9cb44c63571ee1264f53e4b39609c10a","time":{"start":1690292034668,"stop":1690292034668,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55982e1bf4e3 <unknown>\n#1 0x55982deeec76 <unknown>\n#2 0x55982df17d78 <unknown>\n#3 0x55982df14029 <unknown>\n#4 0x55982df52ccc <unknown>\n#5 0x55982df5247f <unknown>\n#6 0x55982df49de3 <unknown>\n#7 0x55982df1f2dd <unknown>\n#8 0x55982df2034e <unknown>\n#9 0x55982e17f3e4 <unknown>\n#10 0x55982e1833d7 <unknown>\n#11 0x55982e18db20 <unknown>\n#12 0x55982e184023 <unknown>\n#13 0x55982e1521aa <unknown>\n#14 0x55982e1a86b8 <unknown>\n#15 0x55982e1a8847 <unknown>\n#16 0x55982e1b8243 <unknown>\n#17 0x7f2e8c273b43 <unknown>","statusTrace":"E   selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n      (unknown error: DevToolsActivePort file doesn't exist)\n      (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\n    Stacktrace:\n    #0 0x55982e1bf4e3 <unknown>\n    #1 0x55982deeec76 <unknown>\n    #2 0x55982df17d78 <unknown>\n    #3 0x55982df14029 <unknown>\n    #4 0x55982df52ccc <unknown>\n    #5 0x55982df5247f <unknown>\n    #6 0x55982df49de3 <unknown>\n    #7 0x55982df1f2dd <unknown>\n    #8 0x55982df2034e <unknown>\n    #9 0x55982e17f3e4 <unknown>\n    #10 0x55982e1833d7 <unknown>\n    #11 0x55982e18db20 <unknown>\n    #12 0x55982e184023 <unknown>\n    #13 0x55982e1521aa <unknown>\n    #14 0x55982e1a86b8 <unknown>\n    #15 0x55982e1a8847 <unknown>\n    #16 0x55982e1b8243 <unknown>\n    #17 0x7f2e8c273b43 <unknown>","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1690292034669,"stop":1690292034694,"duration":25},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"browser","time":{"start":1690292034695,"stop":1690292036657,"duration":1962},"status":"broken","statusMessage":"selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.\n  (unknown error: DevToolsActivePort file doesn't exist)\n  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)\nStacktrace:\n#0 0x55982e1bf4e3 <unknown>\n#1 0x55982deeec76 <unknown>\n#2 0x55982df17d78 <unknown>\n#3 0x55982df14029 <unknown>\n#4 0x55982df52ccc <unknown>\n#5 0x55982df5247f <unknown>\n#6 0x55982df49de3 <unknown>\n#7 0x55982df1f2dd <unknown>\n#8 0x55982df2034e <unknown>\n#9 0x55982e17f3e4 <unknown>\n#10 0x55982e1833d7 <unknown>\n#11 0x55982e18db20 <unknown>\n#12 0x55982e184023 <unknown>\n#13 0x55982e1521aa <unknown>\n#14 0x55982e1a86b8 <unknown>\n#15 0x55982e1a8847 <unknown>\n#16 0x55982e1b8243 <unknown>\n#17 0x7f2e8c273b43 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 80, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/UI-test-automation/UI-test-automation/conftest.py\", line 32, in browser\n    browser = webdriver.Chrome(\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/chrome/webdriver.py\", line 49, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/chromium/webdriver.py\", line 54, in __init__\n    super().__init__(\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 206, in __init__\n    self.start_session(capabilities)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 291, in start_session\n    response = self.execute(Command.NEW_SESSION, caps)[\"value\"]\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/webdriver.py\", line 346, in execute\n    self.error_handler.check_response(response)\n  File \"/opt/hostedtoolcache/Python/3.10.12/x64/lib/python3.10/site-packages/selenium/webdriver/remote/errorhandler.py\", line 245, in check_response\n    raise exception_class(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true}],"afterStages":[],"labels":[{"name":"story","value":"Тест отправления сообщения с контактами."},{"name":"feature","value":"Тесты формы отправления сообщения."},{"name":"tag","value":"contact_message_form"},{"name":"tag","value":"contact_message_tests(scope='class', autouse=True)"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_contact_message"},{"name":"subSuite","value":"TestContactMessageForm"},{"name":"host","value":"fv-az615-165"},{"name":"thread","value":"2343-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_contact_message"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":1,"skipped":0,"passed":1,"unknown":0,"total":2},"items":[{"uid":"156832f12f860966","reportUrl":"https://IlyaYaP.github.io/UI-test-automation/13//#testresult/156832f12f860966","status":"passed","time":{"start":1689967871607,"stop":1689967872619,"duration":1012}}]},"tags":["contact_message_form","contact_message_tests(scope='class', autouse=True)"]},"source":"15d795e7090f9b9e.json","parameterValues":[]}